client/configs/axios.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios'
+
+export const axiosInstance = axios.create({
+    baseURL: 'http://localhost:4000'
```

Comment:
use environment variables to specify the base URL, [refer this doc](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables)